doc/rados/operations/control.rst

```diff
@@ -60,7 +60,7 @@ Here the value of ``[id]`` can be found by consulting the output of ``ceph
 Authentication Subsystem
 ========================
 
-To add an OSD keyring for a specific OSD, run the following command:
+To add an OSD keyring to a specific OSD, run the following command:
```

Comment:
I think this is correct as it stands.  `ceph auth` adds the keyring to the mons.

Reply:
Good catch. Thanks.

Accepted.